src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -228,8 +222,10 @@
         scopes: ALL_SCOPES,
         client_config: {},
         timeout: DEFAULT_TIMEOUT,
-        app_name: "gax",
-        app_version: Google::Gax::VERSION
+        app_name: nil,
+        app_version: nil,
+        lib_name: "gapic",
+        lib_version: "0.1.0"
```

Comment:
This seems like it is probably wrong (either that or _my_ implementation is wrong, which is totally possible!):

  * I was of the understanding that `lib_name` is supposed to be the library built _on top of_ the GAPIC, not the GAPIC itself. If another library is used, _both_ the `gccl` header _and_ the `gapic` header should be sent, correct?
  * `lib_version` probably should not be hard-coded.